pkg/controller/applicationlayer/applicationlayer_controller.go

```diff
@@ -172,77 +179,87 @@ func (r *ReconcileApplicationLayer) Reconcile(ctx context.Context, request recon
 			r.status.OnCRNotFound()
 			return reconcile.Result{}, nil
 		}
-		reqLogger.Error(err, "Error querying for Application Layer")
-		r.status.SetDegraded("Error querying for Application Layer", err.Error())
+		status.SetDegraded(r.status, operatorv1.ResourceReadError, "Error querying for Application Layer", err, reqLogger)
 		return reconcile.Result{}, err
 	}
 	r.status.OnCRFound()
+	// SetMetaData in the TigeraStatus like observedGenerations.
```

Comment:
"like" -> "such as" 
This phrase is used ~10x in this PR.

Reply:
done